src/context/MarFashionProvider.tsx

```diff
@@ -25,7 +25,6 @@ export const MarFashionContext = createContext(defaultState)
 const MarFashionProvider: FC<Props> = ({ children }) => {
   const router = useRouter()
   const [cookies, setCookie, removeCookie] = useCookies(["user", "token"]);
```

Comment:
set default token as null or undefined or empty string

Reply:
done set default values as null for token cookie